src/scripts/activateThebelab.js

```diff
@@ -34,6 +34,11 @@ const getConfig = (language) => {
   };
 
   switch (language) {
+    case 'cpp':
+      config.binderOptions.repo = 'jupyter-xeus/xeus-cling';
+      config.binderOptions.ref = 'stable';
```

Comment:
I chose the stable branch because that is what try.jupyter.org is using.

Reply:
makes sense, I meant to push to a new branch to test, and have reverted it to the previous version